proto/user-service.proto

```diff
@@ -52,6 +64,20 @@ message UpdateUserResponse {
     User user = 1;
 }
 
+message ValidateTokenResponse {
+    bool valid = 1;
+}
+
+message LoginRequest {
+    string username = 1;
```

Comment:
Why does LoginRequest and LogoutRequest require both a token and a username?
Shouldn't it only be the token that is required, and then the token tells us what userid it belongs to?